src/runtime_intrinsics.c

```diff
@@ -1174,9 +1174,123 @@ bi_fintrinsic(mul,mul_float)
 bi_fintrinsic(div,div_float)
 bi_fintrinsic(frem,rem_float)
 
+// runtime fma is broken on windows, define julia_fma(f) ourself with fma_emulated as reference.
+#if defined(_OS_WINDOWS_)
+typedef union {
+  double vf64;
+  uint64_t vu64;
+} ieee_double;
+// reinterpret(UInt64, d::Float64)
+#define BITCAST_F64_U64(ix,d)	\
+do {							\
+  ieee_double temp;				\
+  temp.vf64 = (d);				\
+  (ix) = temp.vu64;				\
+} while (0)
+// reinterpret(Float64, ix::UInt64)
+#define BITCAST_U64_F64(d,ix)	\
+do {							\
+    ieee_double temp;			\
+    temp.vu64 = (ix);			\
+    (d) = temp.vf64;			\
```

Comment:
I believe this is UB. I think the correct way to do this is with a `memcpy` (see https://stackoverflow.com/a/3267062/5141328)

Reply:
I borrowed this pattern from openlibm: https://github.com/JuliaMath/openlibm/blob/master/src/math_private.h#L66-L91.
I‘ll replace it with `memcpy` if that's prefered. 
BTW, thanks again for the julia version!

Reply:
@vtjnash you know this stuff better than me. Is `openlibm` doing a bad here?

Reply:
Using a union isn't UB.